Cargo.toml

```diff
@@ -23,6 +23,8 @@ default = ["alloc"]
 # disable the alloc based ringbuffer, to make RingBuffers work in no_alloc environments
 alloc = []
 
+lending-iterator = []
```

Comment:
Not sure this should be here

Reply:
oh oof, yea it shouldn't. Some experimentation haha